RoslynPluginGenerator/CommandLine/ArgumentProcessor.cs

```diff
@@ -49,6 +49,26 @@ static ArgumentProcessor()
 
         #endregion Argument definitions
 
+        private class NuGetReference
```

Comment:
Was previously public. No longer referenced from anywhere except inside this class.